doc/coding-conventions.xml

```diff
@@ -254,10 +254,13 @@ bound to the variable name <varname>e2fsprogs</varname> in
   dash) — e.g., <literal>"hello-0.3.1rc2"</literal>.</para></listitem>
 
   <listitem><para>If a package is not a release but a commit from a repository, then
-  the version part of the name <emphasis>must</emphasis> be the date of that 
+  the version part of the name should be the date of that 
```

Comment:
Yeah, that makes sense. The date is sufficient for a version. And I suppose the case for `1.0pre` as described in L261 happens when the pre-release cycle spans over several weeks.